### PR TITLE
Italian translation

### PR DESCRIPTION
**Pull request by ElCresh**
Italian translation for app and widget

### CODE REVIEW

File: app/src/main/res/values-it/strings.xml [hunk 1]
```diff
@@ -0,0 +1,186 @@
...
+    <string name="pref_debug_log_summary">Stampa messagi di debug su log.</string>
+    <string name="raspi_auth_keyfile_hint">Private key (formato PKCS8/OpenSSH)</string>
+    <string name="title_placeholders">Segnaposti</string>
+    <string name="update_successful">PI aggioranto con successo.</string>
+    <string name="new_pi_created">PI creato.</string>
+    <string name="permission_private_key_error">Impossibile leggere la primate key. Impossibile eseguire la query.</string>
+    <string name="dialog_passphrase_needed">Passphrase necessaria</string>
+    <string name="ssh_credentials">Credenziali SSH</string>
+    <string name="ssh_credentials">Credenziali SSH</string>
+    <string name="ssh_credentials">Credenziali SSH</string>
```
Comment (eidottermihi) [inline app/src/main/res/values-it/strings.xml]: String `ssh_credentials` is included three times, can you fix it?

Comment (eidottermihi) [reply app/src/main/res/values-it/strings.xml]: I only checked values/strings.xml (default english), there's no duplication there.

**Comment by ElCresh** [comment]
Sorry for delay. I've updated my fork removing duplicated strings.

**Comment by eidottermihi** [comment]
Thank you for contributing!

**Pull request merged**